CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to the versioning scheme outlined in the [README.md](README.md).
 
+## [2.3.0.0.1]
+
+This is a hotfix release to update the peer version identifier used by the stacks-node p2p network.
+
+This release is compatible with chainstate directories from 2.3.0.0.x and 2.1.0.0.x
```

Comment:
I don't think this is necessary since this is not a hard-fork, so the chainstate being compatible is implied.

Reply:
I always opt to put it in the top matter of the release sections in the changelog, just because there _have_ been releases that were consensus-compatible, but also required genesis re-syncs (earlier in the stacks-blockchain's life cycle, but I think it's still a fair question to answer here).

Reply:
+1 for keeping it - even when it's explicit, people ask questions; best to minimize assumptions. That said, at this point anyone not already on 2.2 is already ignored and anyone attempting to boot off a 2.1 chainstate [might run into the issue reported on Discord ](https://discord.com/channels/621759717756370964/730814571517968436/1103320645229617202), so is it worth including a note about that or dropping 2.1 altogether? And include 2.2.0.0.x

Reply:
> I always opt to put it in the top matter of the release sections in the changelog, just because there _have_ been releases that were consensus-compatible, but also required genesis re-syncs (earlier in the stacks-blockchain's life cycle, but I think it's still a fair question to answer here).

I see your point, my reasoning is based almost entirely on historically how the changelog has mentioned chainstate compatibility (seems to be only hard forks had mentioned it). 

I am in favor of *more* info though, but maybe we can adjust so that's clear what the minimum compatible chainstate is? i.e. rather than list a few versions, only list the minimum (in this case 2.1.0.0.0). 

further, i'm in favor of keeping this line for all future versions - but it does add an extra check on us to ensure we have this data.